securedrop/journalist_app/forms.py

```diff
@@ -37,10 +37,17 @@ def name_length_validation(form, field):
             .format(max_chars=Journalist.MAX_NAME_LEN)))
 
 
+def check_invalid_usernames(form, field):
+    invalid_usernames = ['deleted']
+    if field.data in invalid_usernames:
+        raise ValidationError(gettext(
+            "Invalid username '{}'".format(field.data)))
```

Comment:
Let's just leave the message at "Invalid username". The value is visible in the form field immediately above, and not having placeholders simplifies translation.